tests/virt_control_plane_test.go

```diff
@@ -45,14 +45,18 @@ const (
 
 var _ = Describe("[ref_id:2717]KubeVirt control plane resilience", func() {
 
-	RegisterFailHandler(Fail)
+	var err error
+	var virtCli kubecli.KubevirtClient
 
-	virtCli, err := kubecli.GetKubevirtClient()
-	Expect(err).ToNot(HaveOccurred())
-	deploymentsClient := virtCli.AppsV1().Deployments(tests.KubeVirtInstallNamespace)
+	RegisterFailHandler(Fail)
 
 	controlPlaneDeploymentNames := []string{"virt-api", "virt-controller"}
 
+	tests.BeforeAll(func() {
```

Comment:
Why `BeforeAll` and not `BeforeEach` like in the other places? As Roman [mentioned](https://github.com/kubevirt/kubevirt/pull/3786#discussion_r454151180), there are plans to completely remove the `BeforeAll`.

Reply:
Right, `GetKubevirtClient` is safe to be used in `BeforeEach`.
Nice catch!